libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -14,21 +14,31 @@ import type { OperationType } from "@ledgerhq/types-live";
 import { getMainMessage } from "./helpers";
 import { parseAmountStringToNumber } from "./logic";
 
-const getBlankOperation = (tx, fees, id): CosmosOperation => ({
-  id: "",
-  hash: tx.txhash,
-  type: "" as OperationType,
-  value: new BigNumber(0),
-  fee: fees,
-  blockHash: null,
-  blockHeight: tx.height,
-  senders: [] as string[],
-  recipients: [] as string[],
-  accountId: id,
-  date: new Date(tx.timestamp),
-  extra: {},
-  transactionSequenceNumber: parseInt(tx.tx.auth_info.signer_infos[0].sequence),
-});
+const getBlankOperation = (tx: CosmosTx, fees: BigNumber, accountId: string): CosmosOperation => {
+  let transactionSequenceNumber: number;
+
+  try {
+    transactionSequenceNumber = parseInt(tx.tx.auth_info.signer_infos[0].sequence);
+  } catch (e) {
+    transactionSequenceNumber = 0;
+  }
```

Comment:
It's way cleaner to use optional chaining

```suggestion
  const transactionSequenceNumber = parseInt(tx?.tx?.auth_info?.signer_infos[0]?.sequence || "0");
```